### ISSUE TITLE
Jackett not connecting

### ISSUE THREAD

**Issue by BNAS1017** [Jackett/Jackett#1191]
Using Ubuntu 16

Jacket will not connect, and every time I click test it just continues to spin here is the error log:

```
Mar 24, 2017 12:11:56 PM 	Error 	

Unhandled exception url: http://localhost:9117/fonts/glyphicons-halflings-regular.woff2

Mar 24, 2017 12:11:55 PM 	Error 	

Unhandled exception: Message = Object reference not set to an instance of an object
Data = System.Collections.ListDictionaryInternal
InnerException = 
TargetSite = System.Net.IPEndPoint get_RemoteEndPoint()
StackTrace =   at System.Net.HttpConnection.get_RemoteEndPoint () [0x00000] in <5071a6e4a4564e19a2eda0f53e42f9bd>:0 
  at System.Net.HttpListenerRequest.get_RemoteEndPoint () [0x0000b] in <5071a6e4a4564e19a2eda0f53e42f9bd>:0 
  at System.Net.HttpListenerRequest.get_IsLocal () [0x0000b] in <5071a6e4a4564e19a2eda0f53e42f9bd>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerRequest.GetIsLocal () [0x00000] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerContext.GetServerIsLocal () [0x00000] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.get_ServerIsLocal () [0x0000e] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.PropertiesTryGetValue (System.String key, System.Object& value) [0x00156] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.TryGetValue (System.String key, System.Object& value) [0x00000] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.OwinContext.Get[T] (System.String key) [0x00006] in <5decccfc95e44573a9e45d83dbe60e91>:0 
  at System.Web.Http.Owin.OwinHttpRequestContext.get_IsLocal () [0x00008] in <3af1be95d6624fee83df21798930c383>:0 
  at System.Web.Http.Owin.OwinHttpRequestContext.get_IncludeErrorDetail () [0x00037] in <3af1be95d6624fee83df21798930c383>:0 
  at System.Net.Http.HttpRequestMessageExtensions.ShouldIncludeErrorDetail (System.Net.Http.HttpRequestMessage request) [0x00018] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse (System.Net.Http.HttpRequestMessage request, System.Net.HttpStatusCode statusCode, System.Func`2[T,TResult] errorCreator) [0x00007] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse (System.Net.Http.HttpRequestMessage request, System.Net.HttpStatusCode statusCode, System.String message, System.String messageDetail) [0x00014] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Web.Http.Dispatcher.HttpRoutingDispatcher.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) [0x0007f] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Net.Http.DelegatingHandler.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) [0x00006] in <41c316444c8a4380b9344ba09eeb18de>:0 
  at System.Web.Http.Tracing.Tracers.RequestMessageHandlerTracer.<>n__FabricatedMethod6 (System.Net.Http.HttpRequestMessage , System.Threading.CancellationToken ) [0x00000] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Web.Http.Tracing.Tracers.RequestMessageHandlerTracer+<>c__DisplayClass4.<SendAsync>b__1 () [0x00000] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Web.Http.Tracing.ITraceWriterExtensions.TraceBeginEndAsync[TResult] (System.Web.Http.Tracing.ITraceWriter traceWriter, System.Net.Http.HttpRequestMessage request, System.String category, System.Web.Http.Tracing.TraceLevel level, System.String operatorName, System.String operationName, System.Action`1[T] beginTrace, System.Func`1[TResult] execute, System.Action`2[T1,T2] endTrace, System.Action`1[T] errorTrace) [0x00050] in <f99f496cb0d249c1a945c1fcabce1695>:0 
HelpLink = 
Source = System
HResult = -2147467261
```

**Comment by uskyrt** [comment]
+ 1

`Unhandled exception: Message = Object reference not set to an instance of an object
Data = System.Collections.ListDictionaryInternal
InnerException = 
TargetSite = System.Net.IPEndPoint get_RemoteEndPoint()
StackTrace =   at System.Net.HttpConnection.get_RemoteEndPoint () [0x00000] in <5071a6e4a4564e19a2eda0f53e42f9bd>:0 
  at System.Net.HttpListenerRequest.get_RemoteEndPoint () [0x0000b] in <5071a6e4a4564e19a2eda0f53e42f9bd>:0 
  at System.Net.HttpListenerRequest.get_IsLocal () [0x0000b] in <5071a6e4a4564e19a2eda0f53e42f9bd>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerRequest.GetIsLocal () [0x00000] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerContext.GetServerIsLocal () [0x00000] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.get_ServerIsLocal () [0x0000e] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.PropertiesTryGetValue (System.String key, System.Object& value) [0x00156] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.TryGetValue (System.String key, System.Object& value) [0x00000] in <d87779ef62ca4c3698568e9d3e48e24e>:0 
  at Microsoft.Owin.OwinContext.Get[T] (System.String key) [0x00006] in <5decccfc95e44573a9e45d83dbe60e91>:0 
  at System.Web.Http.Owin.OwinHttpRequestContext.get_IsLocal () [0x00008] in <3af1be95d6624fee83df21798930c383>:0 
  at System.Web.Http.Owin.OwinHttpRequestContext.get_IncludeErrorDetail () [0x00037] in <3af1be95d6624fee83df21798930c383>:0 
  at System.Net.Http.HttpRequestMessageExtensions.ShouldIncludeErrorDetail (System.Net.Http.HttpRequestMessage request) [0x00018] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse (System.Net.Http.HttpRequestMessage request, System.Net.HttpStatusCode statusCode, System.Func`2[T,TResult] errorCreator) [0x00007] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse (System.Net.Http.HttpRequestMessage request, System.Net.HttpStatusCode statusCode, System.String message, System.String messageDetail) [0x00014] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Web.Http.Dispatcher.HttpRoutingDispatcher.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) [0x0007f] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Net.Http.DelegatingHandler.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) [0x00006] in <41c316444c8a4380b9344ba09eeb18de>:0 
  at System.Web.Http.Tracing.Tracers.RequestMessageHandlerTracer.<>n__FabricatedMethod6 (System.Net.Http.HttpRequestMessage , System.Threading.CancellationToken ) [0x00000] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Web.Http.Tracing.Tracers.RequestMessageHandlerTracer+<>c__DisplayClass4.<SendAsync>b__1 () [0x00000] in <f99f496cb0d249c1a945c1fcabce1695>:0 
  at System.Web.Http.Tracing.ITraceWriterExtensions.TraceBeginEndAsync[TResult] (System.Web.Http.Tracing.ITraceWriter traceWriter, System.Net.Http.HttpRequestMessage request, System.String category, System.Web.Http.Tracing.TraceLevel level, System.String operatorName, System.String operationName, System.Action`1[T] beginTrace, System.Func`1[TResult] execute, System.Action`2[T1,T2] endTrace, System.Action`1[T] errorTrace) [0x00050] in <f99f496cb0d249c1a945c1fcabce1695>:0 
HelpLink = 
Source = System
HResult = -2147467261`

**Comment by kaso17** [comment]
is this still a problem?
Which mono version are you using?

**Comment by uskyrt** [comment]
Yesterday the New mono app version on my NAS has been made available 4.6.2.r6 
So far no errors anymore. I've got a reminder and I'll report back in 1 week.

**Comment by uskyrt** [comment]
Nope, not happening anymore to me.
